transport_versioned_docs/version-2.0.0/migration.md

```diff
@@ -1,9 +1,46 @@
 ---
 id: migration
-title: migration from 1.x.x
+title: Migration from 1.X
+description: How to deal with breaking changes introduced in version 2.0 of Unity Transport.
 ---
-:::note
-New content -  [MTT-4370](https://jira.unity3d.com/browse/MTT-4370)
-:::
 
+This section describes the breaking changes introduced in version 2.0 of the Unity Transport Package (UTP) and how to update projects written for version 1.X.
 
+Note that for most use cases, no action should be required when updating to 2.0. The core APIs like `NetworkDriver` remain the same, with the biggest changes limited to specialized scenarios (e.g. custom network interfaces).
+
+## Protocol incompatibility
+
+The custom communication protocol used by UTP to implement connections over UDP has changed in a backward-incompatible manner. This means that clients running UTP 2.0 or later can't connect to servers running 1.X, and vice versa. Attempting to establish such connections will result in connection failure after the usual timeout.
+
+It is thus recommended to ensure that clients and servers are updated at the same time, or to disallow older clients from connecting once the server is updated. Another strategy is to offer different endpoints for UTP 1.X and 2.0 servers, which could smooth the transition while older clients are updated.
+
+The reasons for this breaking change are to improve bandwidth efficiency, simplify the protocol, and (ironically) improve forward-compatibility of the protocol.
```

Comment:
The "(Ironically)" piece contrast with the doc. Could we use something like : 
and lays the foundations for better forward compatibility of the protocol.